openarena-q3e

```diff
@@ -1,29 +1,61 @@
 #!/bin/sh
-OA_CVARS="+set fs_basegame baseoa +set protocol 71"
-DISABLED_CVARS="+set com_protocol 72 +set cl_voip 1"
 
+###
+CONFIG_FILE=".var/app/io.github.ec_.Quake3e.OpenArena/config/cfg"
+# Check if the config file exists
+if [ -f "$CONFIG_FILE" ]; then
+    source "$CONFIG_FILE"
+fi
+# Check if it's the first run or if RESPECT_XDGDIRS is not set
+if [ ! -f "$CONFIG_FILE" ] || [ -z "$RESPECT_XDGDIRS" ]; then
+    RESPECT_XDGDIRS=0
+
+    # Save RESPECT_XDGDIRS in the config file
+    echo "RESPECT_XDGDIRS=$RESPECT_XDGDIRS" > "$CONFIG_FILE"
+fi
+# Use the value of RESPECT_XDGDIRS to determine the directory
+if [ "$RESPECT_XDGDIRS" -eq 1 ]; then
+    # Use ${XDG_DATA_HOME}/homepath directory
+    mkdir -p "${XDG_DATA_HOME}/homepath"
+    OA_DIR="${XDG_DATA_HOME}/homepath"
+    if [ -d "$HOME/.openarena" ]; then
+        mv "$HOME/.openarena"/* "$OA_DIR"
```

Comment:
Nice, I see that you now move the `.openarena` directory content before removing it. Thanks!

Reply:
Can we merge this then? I will conduct some testing now. If things go accordingly I'd like to merge this.
On the first run, the user is noticed that he can change the directory, the engine and the game if he wants by pointing the variables to whatever he wants.